index.py

```diff
@@ -81,8 +83,8 @@ def sort_app(organize_by, app_list):
             organized_app_list['All apps'].append(app)
     else:
         for app in app_list:    
-            if app.get(organize_by) is not None:
-                # check if it already exisits in the organized_app_list dictionary.
+            if len(app.get(organize_by)) > 0:
```

Comment:
if `organize_by` doesn't exist, the get will return `None` and this line will fail with a ValueError. You can have it return an optional empty list with `app.get(organize_by, [])` which will still be an okay comparison.